### ISSUE TITLE
Unreadable text with light syntax

### ISSUE THREAD

**Issue by baurigae** [silvestreh/atom-material-ui#18]
Nice theme, and thanks for sharing!
So I'm using a light syntax theme, and have this - the text in this update info box or whatever you call it is very light fg on white bg:
![atmat](https://cloud.githubusercontent.com/assets/5447269/8411797/1facdbb8-1e86-11e5-9480-ca0933661000.png)
I'm not seeing this behavior elsewhere in the preferences.

**Comment by silvestreh** [comment]
Fixed on latest update. 0.4.11 should be available now.

**Comment by baurigae** [comment]
Alright, that fixed it, thanks! and nice work on autocomplete as well, I was just about to complain about that :)
But here's another thing I found:
![r](https://cloud.githubusercontent.com/assets/5447269/8431813/3d7ea20a-1f3c-11e5-90dc-b6da07f56e99.png)
I don't think I should file a separate issue for this?
Anyway this is again a light syntax theme, and I see this is preferences>packages, when I click on settings of a package.
I posted a combo: the top image shows the otherwise barely visible text when selected, the bottom is how it appears under this theme.
